lib/GenomeFileUtil/core/FastaGFFToGenome.py

```diff
@@ -87,6 +87,12 @@ def generate_genome_json(self, params):
 
     def import_file(self, params):
 
+        self.is_metagenome = False
+        ws_datatype = "KBaseGenomes.Genome"
+        if params.get('is_metagenome'):
+            self.is_metagenome = params['is_metagenome']
+            ws_datatype = "KBaseMetagenomes.AnnotatedMetagenomeAssembly"
```

Comment:
sure. But I think it will be more cleaner if you do:
`self.is_metagenome = params.get('is_metagenome', False)
if self.is_metagenome:
 ws_datatype = "KBaseMetagenomes.AnnotatedMetagenomeAssembly"
else:
 ws_datatype = "KBaseGenomes.Genome"`